gensrc/thrift/FrontendService.thrift

```diff
@@ -2121,6 +2121,16 @@ struct TUpdateFailPointResponse {
     1: optional Status.TStatus status;
 }
 
+struct TUpdateTabletVersionRequest {
```

Comment:
Can we TReportRequest to report the tablet version so that reuse most codes on both BE and FE? A difference is that FE should not change tablet meta (add/delete) with this type of report

Reply:
> Can we use TReportRequest to report the tablet version so that reuse most codes on both BE and FE? A difference is that FE should not change tablet meta (add/delete) with this type of report

I previously considered this approach but decided against it for two reasons:

1. ​​Inconsistency with Periodic Report Logic​​
 This behavior differs from the periodic reporting mechanism. Incorporating it into the report logic didn’t align well with the existing reporting flow.
2. ​​Limited Code Reusability​​
Placing it in the report logic didn’t offer significant code reuse. The only reusable part was the leader-checking logic, which wasn’t enough to justify the integration.